english/humbleservant/humbleservant.md

```diff
@@ -0,0 +1 @@
+Turns out that your humble servant is a huge mech robot with a giant drill equipped! Good work assembling that robot before. "My drill is the drill, that creates the HEAVENS", your humble servant shouts, and then it pierces through the marshmallow wall into the stratosphere.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.